packages/form-validations/src/types.ts

```diff
@@ -6,23 +6,12 @@ export interface Languages {
   [key: string]: Language;
 }
 
-// export type Param = {
-//   [key: string]: string;
-// };
-
-// export type Params =
-//   | {
-//       [key: string]: Param;
-//     }
-//   | string;
-
-// export type Params = Record<string, string> | string | never[];
 export type Params = Record<string, string[] | string> | never[];
 
 export type Validator = {
-  fn: (value: string) => boolean | number;
-  priority: number;
-  halt: boolean;
+  fn: (...args: string[]) => boolean | number;
+  priority?: number;
+  halt?: boolean;
   msg?: string | ((message: string, params: Params) => string);
```

Comment:
message?

Reply:
Yes, you are right. I will refactor this, and same the `fn` in the last PR before merge.